### ISSUE TITLE
Sdk out of sync with grafana

### ISSUE THREAD

**Issue by knoguchi** [grafana/grafana-sdk-mocks#9]
I like this sdk-mock idea so I don't have to checkout the whole Grafana tree to develop plugins.
Unfortunately it's out of sync with Grafana main source tree.  For example the PanelCtrl class misses $location property.

Is this sdk-mock official way to develop Grafana plugins?  

The [developer guid](http://docs.grafana.org/plugins/developing/development/) has no mentioning of this node module.  However the [typescript template datasource example](https://github.com/grafana/typescript-template-datasource/blob/master/package.json) linked from the guide uses it.

The sdk-mock has no version tags.    Looks like an experimental project to me.

Created a PR #10 that patches the sdk just to pass my compiling.  I'm sure there are many more missing.  Can the stubs be autogenerated?

**Comment by sra90** [comment]
Yes, many modules like file_export, app_events, etc are missing from this.
